performance.md

```diff
@@ -29,10 +29,10 @@ Global optimizations are used to speed up the overall messages processing improv
 
 Rspamd uses various methods to speed up each individual message processing stage. This is achieved by applying local optimizations techniques:
 
-* **<abbr title="Abstract Syntax Tree">AST</abbr> optimizations** are used to exclude unnecessary checks from rules. You can watch the following [slides](http://www.slideshare.net/VsevolodStakhov/astrspamd) to get more details about this method.
+* **<abbr title="Abstract Syntax Tree">AST</abbr> optimizations** are used to exclude unnecessary checks from rules. You can watch the following [slides](https://www.slideshare.net/VsevolodStakhov/astrspamd) to get more details about this method.
 
 * Unlike SA, Rspamd uses **specific state machines** to parse email components: mime structure, HTML parts, URLs, images, received headers and so on and so forth. This approach allows to skip unnecessary details and extract information from emails quicker than by using a large set of regular expressions for these purposes.
 
-* **[Hyperscan](https://github.com/intel/hyperscan) engine** is used in Rspamd to quickly process large regular expressions set. Unlike traditional regexps engines, Hyperscan allows to process multiple expressions at the same time. There are many details about hyperscan that are covered in the following [slides](http://www.slideshare.net/VsevolodStakhov/rspamdhyperscan).
+* **[Hyperscan](https://github.com/intel/hyperscan) engine** is used in Rspamd to quickly process large regular expressions set. Unlike traditional regexps engines, Hyperscan allows to process multiple expressions at the same time. There are many details about hyperscan that are covered in the following [slides](https://www.slideshare.net/VsevolodStakhov/rspamdhyperscan).
```

Comment:
This page is unavailable: https://www.slideshare.net/VsevolodStakhov/rspamdhyperscan

Reply:
I checked it and it is ;)
Here is the screenshot proof
![Screenshot_2023-07-18-16-18-04-193_org mozilla fenix](https://github.com/rspamd/rspamd.com/assets/7784660/49d73498-af11-4df5-a79d-113ccff86b44)